src/Organizations/index.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from "react";
```

Comment:
```suggestion
import React from "react";
```
Unused variable. We don't need to import useState hook here.